static/posts/how-to-build-a-blog.md

```diff
@@ -0,0 +1,258 @@
+---
+title: How to Build a Blog
+date: 12/10/2023
+tags:
+  - react
+  - deno
+  - fresh
+description: How and why I built my blog ... and you can too!
+coverAltText: "Note: Although this image is AI generated, the rest of this post is not."
+---
+
+## Motivation
+
+Let's say you've always wanted to build your own website. Maybe you're
+interested in learning React. Perhaps you're tired of complicated Node JS web
+frameworks and don't want to spend an exorbitant amount of time learning one on
+top of React. Possibly you're a bit sick of Node JS in general.
+
+I have good news! Many other people feel the same way. Even the creator of Node.
+So much so, that he's created a new JavaScript (and TypeScript) runtime called
+Deno. It fixes many of the things about Node JS that probably leave a bitter
+taste in your mouth. Such as:
+
+- out of the box TypeScript support
+- built-in formatting
+- built-in linting
+- top-level async/await
+- no node_modules/
+- ECMAScript Module support (no CommonJS 😎)
+- ...
+
+These same people, who are tired of cumbersome Node JS web frameworks, have
+created some slick, **minimal**, web frameworks for it. One such framework is
+Fresh. If you're thinking: "No! Not another web framework!", hear me out. Fresh
+is great because out of the box it has:
+
+- no build step
+- no configuration
+- speed
+- simplicity
+
+Fresh does all of those things and more. It's so clean and simple that it "makes
+web development fun again". Deno and Fresh complement each other like sunshine
+and beaches.
```

Comment:
You're so right for using complement instead of compliment. Honestly, I probably would've written it wrong. Goes hard.